bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -599,5 +605,9 @@ public void run() {
 
     Runtime.getRuntime().addShutdownHook(reportingThread);
   }
+
+  public static boolean isAlpnConfigured() {
+    return OpenSsl.isAvailable() || OpenSsl.isAlpnSupported();
```

Comment:
Not sure what this means? By default if OpenSsl.isAvailable() == true, OpenSsl is used, not Alpn.